packages/components/src/components/file-uploader/_file-uploader.scss

```diff
@@ -102,7 +102,7 @@
     margin-bottom: $carbon--spacing-05;
   }
 
-  .#{$prefix}--file-btn ~ .#{$prefix}--file-container {
+  .#{$prefix}--btn ~ .#{$prefix}--file-container {
```

Comment:
I couldn't find where this selector was being used before but would this be a breaking change?

Reply:
Honestly, i'm not really sure. It doesn't seem like it since `.#{$prefix}--file-btn` isn't being used anywhere and the only styles I can find for it say this: 
```
  // This class is of old markup with "select file" button
  // New code should use link-style "select file" UI (`.bx--file-browse-btn`)
  // TODO: deprecate this block
  .#{$prefix}--file-btn {
    display: inline-flex;
    margin: 0;
    padding-right: rem(64px);
  }
```

Reply:
hm yeah since it's marked with a deprecation note I believe it's still there for legacy markup so we probably can't remove it until next major version, (deprecation note added in #2848 and incorporated in #3872 cc @asudoh)

Reply:
Good point, style for legacy markup cannot be removed until the next major release. That said, probably we keep this intact?

Reply:
I added back the old selector along with this new one so there shouldn't be any problem now! 👍